File: eTierMeeting_v2/SPA/src/app/_core/_models/roles.ts
export interface Roles {
  id: number;
  roleName: string;
  updateBy: string;
  updateTime: string | null;
  roleSequence: number | null;
}


export interface RolesDTO {
  id: number;
  roleName: string;
  updateBy: string;
  updateTime: string | null;
  roleSequence: number | null;
  checked: boolean;
  visible: boolean | true;
}

File: eTierMeeting_v2/API/Models/MachineCheckList/Roles.cs

using System.ComponentModel.DataAnnotations;

namespace Machine_API.Models.MachineCheckList
{
    public class Roles
    {
        [Key]
        public int ID { get; set; }
        [Required]
        [StringLength(50)]
        public string RoleName { get; set; }
        [Required]
        [StringLength(20)]
        public string UpdateBy { get; set; }
        public DateTime? UpdateTime { get; set; }
        public int? RoleSequence { get; set; }
    }
}